src/govuk/components/input/template.test.js

```diff
@@ -310,23 +310,23 @@ describe('Input', () => {
 
       const $prefix = $('.govuk-form-group > .govuk-input__wrapper > .govuk-input__prefix')
 
-      expect($prefix.html()).toEqual('&#xA3;')
+      expect($prefix.html()).toEqual('£')
```

Comment:
We definitely didn't have `£` as `&#xA3;` in the HTML

(Confirmation that it was `cheerio@1.0.0-rc.3` encoding things it shouldn't)